client/lib/url/test/http-utils.js

```diff
@@ -0,0 +1,125 @@
+/**
+ * @jest-environment jsdom
+ */
+
+/**
+ * Internal dependencies
+ */
+import { withoutHttp, urlToSlug, urlToDomainAndPath } from '../http-utils';
+
+describe( 'withoutHttp', () => {
+	test( 'should return null if URL is not provided', () => {
+		expect( withoutHttp() ).toBeNull;
```

Comment:
@sgomes Something I noticed in #36939 - are these tests working as expected?

These look like chai expects. All jest expects should be functions:

```js
expect( withoutHttp() ).toBeNull;    // No!
expect( withoutHttp() ).toBeNull();  // Yes :)
```

Reply:
Good spot, @sirreal! I guess it doesn't fail because it doesn't do anything. I'll go ahead and prepare a PR fixing this, thanks!